readthedocs/search/documents.py

```diff
@@ -1,7 +1,7 @@
 import logging
 
 from django.conf import settings
-from django_elasticsearch_dsl import DocType, Index, fields
+from django_elasticsearch_dsl import Document, Index, fields
```

Comment:
I'm curious if we're getting much value from the django integration here. It might make sense to look into just using the elasticsearch_dsl directly. I think we've disabled most of the Django-specific integration, so that might make our code easier to manage and a lot more explicit.

Reply:
Yeah, I saw that too. The main feature of the django integration is that it syncs the ES index when the models change, but we don't make use of that. I'll make a note.